nrel_5mw/g0/nrel_5mw_g0_hypre.i

```diff
@@ -249,25 +254,37 @@ realms:
         search_tolerance: 0.01
         search_method: stk_kdtree
 
+    mesh_motion:
+
+      - name: mesh_motion_rotor
+        mesh_parts:
+          - block_101
+          - block_104
+          - block_105
+          - block_106
+        frame: non_inertial
+        compute_centroid: yes
+        motion:
+          - type: rotation
+            omega: 0.9587301587301587 #TSR=7.55 @ 8.0 m/s inflow velocity
+            axis: [0.9961946980917455, 0.0, -0.08715574274765817]
+
+      - name: mesh_motion_outer_domain
+        mesh_parts:
+          - block_201
+          - block_204
+          - block_205
+          - block_206
+        motion:
+          - type: rotation
+            omega: 0.0
+
     solution_options:
 
       name: myOptions
```

Comment:
Does the name`myOptions` matter?  In other words, would another name be more helpful/relevant?

Reply:
@jhux2 The name can be something descriptive you prefer, it is not used internally for anything.

Reply:
It sounds like the name isn't terribly important, so I'm fine keeping it as is.